ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -123,6 +124,18 @@ public void removeOptimizer(String resourceId) {
     authOptimizers.values().removeIf(op -> op.getResourceId().equals(resourceId));
   }
 
+  public void loadOptimizers() {
+    List<OptimizerInstance> optimizers = getAs(OptimizerMapper.class, OptimizerMapper::selectAll);
+    if (optimizers == null || optimizers.isEmpty()) {
+      return;
+    }
```

Comment:
Loading all optimizers from sysdb for each OptimizingQueue separately is not a good practice.

How about construct the OptimizingQueue with these optimizers loaded in `DefaultOptimizingService`?